MaterialLib/PorousMedium/Permeability/createPermeabilityModel.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   createPermeabilityModel.cpp
+ *
+ * Created on August 17, 2016, 2:36 PM
+ */
+
+#include "createPermeabilityModel.h"
+
+#include <cassert>
+
+#include "BaseLib/Error.h"
+#include "BaseLib/StringTools.h"
+
+namespace MaterialLib
+{
+namespace PorousMedium
+{
+CoefMatrix createPermeabilityModel(BaseLib::ConfigTree const* const config)
+{
+    //! \ogs_file_param{material__porous_medium__permeability__values}
+    auto const val_strs = config->getConfigParameter<std::string>("values");
+    std::vector<double> values =
+        BaseLib::convertStringValues2Values<double>(val_strs);
```

Comment:
This is not needed. ConfigTree can read vectors of space separated values. (just use `<std::vector<double>>` instead of `<std::string>` above.